appveyor.yml

```diff
@@ -41,190 +45,215 @@ install:
 
     nuget install -Verbosity quiet -OutputDirectory packages -Version 1.8.14 Doxygen
 
-    nuget install -Verbosity quiet -OutputDirectory packages secure-file -ExcludeVersion
-
 before_build:
 - ps: >-
     if($env:APPVEYOR_PULL_REQUEST_NUMBER -eq $null)
+
     {
+
       git clone https://$env:CREDENTIALS_GITHUB_TOKEN@github.ibm.com/germanatt/sdk-credentials.git C:\projects\sdk-credentials
+
     }
 
+    $env:IBM_CREDENTIALS_FILE = "C:\projects\sdk-credentials\ibm-credentials.env"
+
     dotnet restore
 
 build:
-  project: IBM.Watson.sln
-  verbosity: minimal
+  project: dotnet-sdk-core.sln
+  verbosity: normal
 after_build:
 - ps: >-
     git config --global user.email "wps@us.ibm.com"
-    git config --global user.name "Watson Github Bot"
+
+    git config --global user.name "IBM Github Bot"
 
     if(!$env:APPVEYOR_REPO_TAG_NAME)
 
     {
+

```

Comment:
This all looks good minus all these newlines 😮

Reply:
Yeah I noticed when I was working in the AppVeyor UI and exported the .yml it had these returns in them. I kept them to be safe since I found at least one case (git config --global) it treated the newline as hitting a return in the script. Before it was running the command as
```
git config --global user.email "wps@us.ibm.com"git config --global user.name "Watson Github Bot"
```

[here](https://ci.appveyor.com/project/mediumTaj/dotnet-sdk-core/builds/23345899#L275) is an example from one of the logs for when the newline isn't there.